query-algebrizer/src/cc.rs

```diff
@@ -115,13 +128,29 @@ pub fn default_table_aliaser() -> TableAliaser {
     })
 }
 
-#[derive(PartialEq, Eq, Debug)]
+#[derive(PartialEq, Eq)]
 pub enum ColumnConstraint {
     EqualsEntity(QualifiedAlias, Entid),
     EqualsValue(QualifiedAlias, TypedValue),
     EqualsColumn(QualifiedAlias, QualifiedAlias),
 }
 
+impl fmt::Debug for ColumnConstraint {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            &ColumnConstraint::EqualsEntity(ref qa, ref entid) => {
+                write!(f, "cc::Entity({}/{:?} = {:?})", qa.0, qa.1, entid)
```

Comment:
I'd prefer these be:

```rust
   write!(f, "{}.{} = entity({:?})", qa.0, qa.1.as_str(), entid)
```

and so on — instead of

```
    cc::Entity(datoms00/Attribute = 98),
    cc::Column(datoms01/Value = datoms02/Entity),
```

it would be


```
    datoms00.a = entity(98),
    datoms01.v = datoms02.e,
```

which is much closer to the syntax we use in example SQL, and a little less crusted in syntax.